com.unity.multiplayer.mlapi/Editor/ProfilerModuleOverride/NetworkProfilerOverride.cs

```diff
@@ -0,0 +1,249 @@
+using System;
+using System.Collections.Generic;
+using MLAPI.Profiling;
+using Unity.Profiling;
+using UnityEditor;
+using UnityEngine;
+
+namespace ProfilerModuleOverride
+{
+    [InitializeOnLoad]
+    public class NetworkProfilerOverride
+    {
+#if UNITY_2020_2_OR_NEWER
+        const string RPCModuleName = "MLAPI RPCs";
+        const string OperationModuleName = "MLAPI Operations";
+        const string MessageModuleName = "MLAPI Messages";
+
+        [System.Serializable]
+        public class MLAPIProfilerCounter
+        {
+            public string m_Name;
```

Comment:
nit, `m_` prefix is meant only for private members

Reply:
This has to be named this way because of the json serialization has to be the same name as those in the c++ code which uses m_

Reply:
Ohh. We should add some comments here then. That is not clear at all from the code that this is being deserialized by another unrelated codebase

Reply:
Agreed I will put together a little comment about what this is actually doing.

Reply:
Comment has been added